### DISCUSSION TITLE
I am correct to understand that such CVE's are being patched to magento-lts

### DISCUSSION THREAD

**Discussion by f1-outsourcing** [OpenMage/magento-lts#1789] [Questions & Answers]
Sorry for asking maybe the obvious, but Am I correct to understand that such CVE's are being patched to magento-lts, is there some changelog or so that mentions these?

https://cve.mitre.org/cgi-bin/cvekey.cgi?keyword=magento

**Comment by fballiano** [comment]
yes, but anyway most of them are related to M2 ;-)

**Reply by f1-outsourcing** [reply]
I know, I know!!!  In a discussion on reddit I even stated that I suspected that there would be more exploits in M2, because the development/management team look to me like rookies, no vision no clear strategy.
